app/services/otp_rate_limiter.rb

```diff
@@ -41,9 +41,11 @@ def increment
 
   attr_reader :phone, :user, :phone_confirmed
 
+  # rubocop:disable Naming/MemoizedInstanceVariableName
   def entry_for_current_phone
     @entry ||= OtpRequestsTracker.find_or_create_with_phone_and_confirmed(phone, phone_confirmed)
```

Comment:
I could have renamed the ivar in `#increment` too but this seemed easier to leave as-is